code/modules/VR/vr_avatar.dm

```diff
@@ -0,0 +1,87 @@
+
+
+/mob/living/carbon/human/virtual_reality
+	var/mob/living/carbon/human/real_me //The human controlling us, can be any human (including virtual ones... inception...)
```

Comment:
With this you just need to define a map template, an outfit, and a map_template entry and it will show up in the useable rooms list. Plus rooms can be reset and multiple copies can be made.

Also dealing with duplicates was a pain. Right now I just have it search the whole space chunk for every land mark in the game and return a list of thoes in the chunk. Its messy but it works I guess. I am super new to coding, this is my second real PR.